.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+      - release/v*
+
+jobs:
+  bump-version:
+    name: Bump secrethub.ClientVersion
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Bump version
+        uses: florisvdg/action-version-bump@master
```

Comment:
Shall we use a pinned version?

Reply:
I didn't create a release for it yet, but when I do I'll pin it here. But seeing as it's not a critical CI/CD component (you can always set it manually) I'll leave it like this for now.

Reply:
Our branch protection policy doesn't allow a branch to be merged when any of the checks fails. Can we set it optional somehow?

Reply:
Is there a reason not to release a version yet?

Reply:
It's excluded: 
![image](https://user-images.githubusercontent.com/7430639/80809073-3b6a6980-8bc1-11ea-9fcc-3df37635b5e0.png)

Reply:
Yes, that means if no status is reported at all, that's fine.
However, if the action reports a failure status, the PR isn't allowed to be merged.

Reply:
Okay pinned it.